### DISCUSSION TITLE
Feature Request: Add Reddit to supported social icons

### DISCUSSION THREAD

**Discussion by Ahmed-E-86** [chipzoller/hugo-clarity#194] [Q&A]
I was testing hugo-clarity with my blog, and I added Reddit to social, but its icon did not appear. I checked icons.html, and I found out that Reddit it is not supported.

Are there any intentions to add it?

**Comment by onweru** [comment]
@Ahmed-E-86 

> Are there any intentions to add it?

Not really, we add icons on a request/need-basis. You can add the reddit icon like so https://github.com/chipzoller/hugo-clarity/pull/110/commits
